cdap-docs/reference-manual/build.sh

```diff
@@ -70,12 +70,19 @@ function build_extras() {
     echo "Not using Javadocs."
   fi
 
-  cp ${SCRIPT_PATH}/licenses-pdf/*.pdf ${TARGET_PATH}/html/licenses
-  warnings=$?
-  if [[ ${warnings} -eq 0 ]]; then
-    echo "Copied license PDFs."
+  if [ -d ${TARGET_PATH}/html/licenses ]; then
+    cp ${SCRIPT_PATH}/licenses-pdf/*.pdf ${TARGET_PATH}/html/licenses
+    warnings=$?
+    if [[ ${warnings} -eq 0 ]]; then
+      echo "Copied license PDFs."
+    else
+      local m="Error ${warnings} copying license PDFs."
+      echo_red_bold "${m}"
+      set_message "${m}"
+      return ${warnings}
+    fi
   else
-    local m="Error ${warnings} copying license PDFs."
+    local m="Error TARGET_PATH ${TARGET_PATH}/html/licenses does not exist."
```

Comment:
Yes, it does. That's what I meant about you would still get some errors, even if the build was clean. Here's typical output:
```
--------------------------------------------------------
Warning Messages: .build.sh.50201.messages
--------------------------------------------------------


Warning Messages for "reference-manual":
--------------------------------------------------------

Error 2 building CLI docs table

Error in download_includes: 2

Error TARGET_PATH /Users/john/Source/cdap/cdap-docs/reference-manual/target/html/licenses does not exist.

--------------------------------------------------------
End Warning Messages
--------------------------------------------------------

--------------------------------------------------------
Completed "Building Docs Only: docs_only"
========================================================
========================================================
```
If this is not acceptable, I can find a way to remove these...